deploy/terraform-custom-datacommons/README.md

```diff
@@ -0,0 +1,197 @@
+# Custom Data Commons Terraform Deployment
+
+## Overview
+
+Deploying your own Custom Data Commons instance on Google Cloud Platform (GCP) lets you to host and manage your own data while leveraging [Google's Data Commons](https://datacommons.org/) repository of publicly available information. This deployment is managed using Terraform, which automates the provisioning of infrastructure and services on GCP.
```

Comment:
"lets you host" or "allows you to host"?

Reply:
Fixed